lib/cdo/contact_rollups.rb

```diff
@@ -886,7 +895,21 @@ def self.update_district
     # has district names like "open csp".
     districts = District.all.index_by(&:id)
 
-    users = User.where("length(email) > 0")
+    # MULTIAUTH edit point
+    # users = User.where("length(email) > 0") # original query, replaced by following
+    users = User.find_by_sql(<<-eos
```

Comment:
Dumb Ruby question: I thought you finish the expression around the HEREDOC opener?  Or is that optional?

```